src/components/00-materials/icons-raw/download_outlined.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M13 5v6h1.17L12 13.17 9.83 11H11V5h2m2-2H9v6H5l7 7 7-7h-4V3zm4 15H5v2h14v-2z" fill="currentColor"/></svg>
```

Comment:
`fill="none"` sure?

Reply:
Yes, must be. I had it on `fill="currentColor"` and removed it completely. This is the only way it works.